code/modules/research/designs.dm

```diff
@@ -604,7 +588,45 @@
 	build_path = /obj/item/weapon/gun/energy/cutter/plasma
 	sort_string = "TAEAC"
 
-
+/datum/design/item/weapon/divet
+	name = "Winchester NK Divet pistol"
+	id = "divet"
+	req_tech = list(TECH_COMBAT = 1, TECH_MATERIAL = 3)
```

Comment:
Make this combat tech 2, as per our previous conversation. Not sure if you glanced over it or not.